client/src/components/teamPage/TeamMember.js

```diff
@@ -94,11 +101,11 @@ const TeamMember = ({ member }) => {
         </Card>
       </Col>
 
-      <Modal show={showUpdateForm}>
-        <Modal.Header className="modalHeader">
-          <Modal.Title> Edit Team Member </Modal.Title>
+      <Modal show={showUpdateForm} id="teamMemberModal">
+        <Modal.Header className="teamMemberModalHeader">
```

Comment:
We don't currently any CSS class for `teamMemberModalHeader`, so this would have no effect right?

Reply:
Good catch! Forgot to add this class but basically just needed the background colour for this to be white and adding in a non-existent class changed the background colour from grey to white, so didn't realise we forgot to add this. Have added this in now.